packages/ses/index.d.ts

```diff
@@ -69,8 +69,14 @@ export interface RedirectStaticModuleInterface {
   importMeta?: any;
 }
 
+export interface AliasModuleInterface {
+  alias: string;
+  compartment: Compartment;
```

Comment:
why would the alias come from a different compartment if the goal for this is to support the `browser` field? 
is this the part where browser field defines a different package name for an import? I was thinking of the browser field pointing to a different export of a package. 
Why would an importHook know of a different compartment than its own?

TL;DR: what's the point of having `compartment` here?

Reply:
This feature should be sufficiently general to also handle `imports` in `package.json`.